packages/kbn-apm-synthtrace/src/cli/utils/exec.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { spawn } from 'child_process';
+
+export async function exec(shellCommand: string, env?: Record<string, string>): Promise<string> {
```

Comment:
Do we need a custom implementation for this? isn't there something that we can re-use? What's the difference between this and child_process.exec?